index.html

```diff
@@ -28,7 +28,7 @@
                     <li class="nav-item">
                         <a href="#">Contact</a>
                     </li>
-                    <li class="nav-item">
+                    <li id="hidemail" class="nav-item">
                         <a href="#"><img src="./assets/Disabled.png" alt="mail"></a>
```

Comment:
- [ ] please ensure that the close button is with the mobile popup menu and not separated from it. when scrolling on your mobile menu the close button separates from the other links.
your version 👇🏾
![Screenshot (123)_LI](https://user-images.githubusercontent.com/65298048/173634017-2b9e2869-cd37-40e6-8eea-0f7fd4bf59c6.jpg)

Reply:
Made Changes to keep the hamburger menu icon fixed.